.github/workflows/ci.yml

```diff
@@ -5,8 +5,13 @@ on:
   pull_request:
   push:    
     branches:
-      - main
-           
+      - ci_fix
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
 jobs:      
   test:    
     name: molecule
```

Comment:
Let's change the name since there is no Molecule being performed. If we want to add it later on, we can always rename the job back